### PR TITLE
Updated container access documentation

### PR DESCRIPTION
**Pull request by janjiss**
ROM.container is not working, since the container in rails is located under ```ROM::Rails::Railtie.container```

### CODE REVIEW

**Comment by cflipse** [comment]
That's not how we want to actually access the thing though.

`ROM.env` currently works.

Also, within a controller, `rom` _should_ work


alternatively, we'll be working on fixing what's in ROM-Rails so that this becomes correct, though that will require a new release.

**Comment by janjiss** [comment]
@cflipse Updated docs. I can do a writeup about controller bit later if you don't mind.

**Comment by AMHOL** [comment]
:+1: Nice one @janjiss

File: source/learn/getting-started/rails-setup.html.md [hunk 1]
```diff
@@ -70,6 +70,15 @@ In Rails environment ROM container is accessible via `ROM.env`:
 ROM.env # returns the container
 ```
 
+In your controllers you can access ROM container simply like this:
+``` ruby
+class HomesController < ApplicationController
+  def show
+    rom #=> returns rom container
```
Comment (cflipse) [inline source/learn/getting-started/rails-setup.html.md]: Maybe show this doing something valid in a rails context?  For some reason, I find the blankness of it distracting.
```
@home = rom.relation(:homes).by_id(params[:id]).one
```

Comment (janjiss) [inline source/learn/getting-started/rails-setup.html.md]: Ok, that makes sense. I'll think of example and create another PR.

**Comment by janjiss** [comment]
@cflipse Does this look better? My concern is that this example is shown before actual relations are set up? I can move it to the bottom maybe?

**Comment by cflipse** [comment]
Yep, looks good to me.

It's probably a fair point that we may be showing things that havn't quite been covered yet; if that's an issue for understanding, I think we can move the "accessing" section lower under a separate PR.  I'm going to go ahead and merge this one in now.

Thanks!

**Pull request merged**